src/common/paddle/slack.ts

```diff
@@ -278,3 +279,137 @@ export const notifyNewPaddlePlusTransaction = async ({
 
   await webhooks.transactions.send({ blocks });
 };
+
+export const notifyNewPaddleOrganizationTransaction = async ({
```

Comment:
Shouldn't this have a early return if it's not a org one?
I don't see it would only fire for those

Reply:
There is `findOneByOrFail`, so if no org is found, it will fail